src/test/e2e/99_yolo_test.go

```diff
@@ -47,3 +47,17 @@ func TestYOLOMode(t *testing.T) {
 	stdOut, stdErr, err = e2e.Zarf("package", "remove", "yolo", "--confirm")
 	require.NoError(t, err, stdOut, stdErr)
 }
+
+func TestDevDeploy(t *testing.T) {
+	// Don't run this test in appliance mode
+	if e2e.ApplianceMode {
+		return
+	}
+	e2e.SetupWithCluster(t)
+
+	stdOut, stdErr, err := e2e.Zarf("dev", "deploy", "examples/yolo")
```

Comment:
```suggestion
	stdOut, stdErr, err := e2e.Zarf("dev", "deploy", "examples/dos-games")
```
We should test with a more realistic package to make sure we are _really_ in yolo mode